playbooks/robusta_playbooks/krr.py

```diff
@@ -140,14 +138,14 @@ def krr_scan(event: ExecutionBaseEvent, params: KRRParams):
     """
     Displays a KRR scan report.
     """
-
+    header = PrometheusAuthorization.get_authorization_headers(params)["Authorization"]
     spec = PodSpec(
         serviceAccountName=params.serviceAccountName,
         containers=[
             Container(
                 name=to_kubernetes_name(IMAGE),
                 image=IMAGE,
-                command=["/bin/sh", "-c", f"python krr.py {params.strategy} {params.args_sanitized} -q -f json"],
+                command=["/bin/sh", "-c", f"python krr.py {params.strategy} {params.args_sanitized} -p {params.prometheus_url} --prometheus-auth-header \"{header}\" -q -f json"],
```

Comment:
The command is part of the job spec.
I think we need to pass the `auth-header` in a secure manner.